src/win/platform_api.rs

```diff
@@ -1,13 +1,15 @@
 use std::path::{Path, PathBuf};
+use std::str::FromStr;
 
+use windows::Win32::System::ProcessStatus::K32GetProcessImageFileNameA;
```

Comment:
```suggestion
use windows::Win32::System::ProcessStatus::K32GetProcessImageFileNameW;
```

Use `W` (unicode) functions instead of `A` (ANSI) in win api.

Reply:
Implemented in 1699bcf